test/integration.html

```diff
@@ -12,8 +12,20 @@
 <head>
   <title>iron-location</title>
 
-  <script src="../../webcomponentsjs/webcomponents.js"></script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
   <script src="../../web-component-tester/browser.js"></script>
+  <script>
```

Comment:
this script is not needed as URL polyfill already checks for it https://github.com/webcomponents/URL/blob/master/url.js#L13